gradle-jarjar-plugin/jarjar/build.gradle

```diff
@@ -0,0 +1,27 @@
+apply plugin: 'groovy'
+
+dependencies {
+    compile gradleApi()
+    compile localGroovy()
+}
+
+task javadocJar(type: Jar) {
+    classifier = 'javadoc'
+    from javadoc
+}
+
+task sourcesJar(type: Jar) {
+    classifier = 'sources'
+    from sourceSets.main.allSource
+}
+
+artifacts {
+    archives javadocJar, sourcesJar
+}
+
+
+def isReleaseBuild() {
+    return VERSION_NAME.contains("SNAPSHOT") == false
```

Comment:
This can be simplified `!VERSION_NAME.contains("SNAPSHOT")`

Reply:
done